internal/parser/experiment.go

```diff
@@ -19,6 +19,7 @@ type inputExperiment struct {
 	WorldSize  inputWorldSize
 	Targets    []string
 	Warmup     uint
+	Automator  any `default:"three_nodes.pos"`
```

Comment:
This shouldn't be the default value. In fact, I don't think an default value is needed after all. Just leave it as string and it will be `""` if the user doesn't specify any automator.